src/Platform/Hosted/extractFeatures.ts

```diff
@@ -37,6 +37,7 @@ export type Features = {
   phoenixNotebooks?: boolean;
   phoenixFeatures?: boolean;
   notebooksDownBanner: boolean;
+  publicGallery?: boolean;
```

Comment:
Do you need to add support for this flag in the method below? Otherwise this can't be used as feature flag

Reply:
No this is set through flighting, we dont want to expose this through feature flags otherwise it could be misused